src/ja/server/config.py

```diff
@@ -1,7 +1,37 @@
+from typing import Dict
+
 from ja.common.config import Config
 
 
 class ServerConfig(Config):
```

Comment:
Several missing configuration options:

1. Amount of special resouces (number of licenses, etc)
2. Server port?
3. Database connection information? Or where should it come from?

Reply:
What exactly do you mean by database connection information?
Is it specified somewhere in the code?

Reply:
I am not very sure actually, because I am not very familiar witth databases.
But I assumed you do have some sort of username/password or at least a database identifier, right?

Reply:
I don't know if we're going to use PostreSQL, but just an example: https://wiki.postgresql.org/wiki/Using_psycopg2_with_PostgreSQL

There, you need a host, database name, user name and password to connect to the database.

Reply:
Or maybe we'll hardcode them?

Reply:
I think most of the times a configuration file. But we can also hardcode them.

Reply:
I'll just add another config.
It's not much work (both design and implementation).

Reply:
What are we going to need a server port for?

Reply:
Oops, didn't see this when merging. Basically, we need a port for the web api.